helper/EventHelper.go

```diff
@@ -170,6 +176,58 @@ type CommonWorkflowRequest struct {
 	CiArtifactLastFetch      time.Time     `json:"ciArtifactLastFetch"`
 }
 
+func (c *CommonWorkflowRequest) GetCloudHelperBaseConfig(blobStorageObjectType string) *util.CloudHelperBaseConfig {
+	return &util.CloudHelperBaseConfig{
+		StorageModuleConfigured: c.BlobStorageConfigured,
+		BlobStorageLogKey:       c.BlobStorageLogsKey,
+		CloudProvider:           c.CloudProvider,
+		UseExternalClusterBlob:  c.UseExternalClusterBlob,
+		BlobStorageS3Config:     c.BlobStorageS3Config,
+		AzureBlobConfig:         c.AzureBlobConfig,
+		GcpBlobConfig:           c.GcpBlobConfig,
+		BlobStorageObjectType:   blobStorageObjectType,
+	}
+}
+
+type BlobStorageConfig struct {
```

Comment:
Please add some comments explaining why we need it here as well, when it was also coming from orchestrator

Reply:
done